### PR TITLE
Feature popup

### PR DESCRIPTION
**Pull request by Spacelama587**
Features Implemented

When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.
All information in the popup window is is created dynamically using information stored in Javascript array

### CODE REVIEW

**Comment by Spacelama587** [comment]
SS

**Review by joel-kalema — COMMENTED** [review]
Hi Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 

pirates-of-the-caribbean-johnny-depp


- [ ] You have submitted a project that your partner didn't contribute to. This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#work-and-submission-mode).


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @Spacelama587   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉

- [x] No linter errors. ✔️

- [x] Github flow is followed. ✔️

- [x] Worked on some requirements 
- [x] Desriptive PR
- [x] Professional README
- [x] Nice UI design

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -115,253 +114,7 @@ <h4 >
 
   </main>
```
Comment (divinecharlotte) [inline index.html:116]: - [ ] kindly consider making sure that you have an element here where you are displaying your works-section in your .js file using DOM manipulation to make that section visible on your website cause current its not showing up

File: script.js [hunk 2]
```diff
@@ -21,3 +21,508 @@ menuLinks.forEach((link) => {
     logo.classList.remove('hide-logo');
   });
 });
+
+document.addEventListener('DOMContentLoaded', () => {
+  const section = document.createElement('section');
+  section.className = 'works';
```
Comment (divinecharlotte) [inline script.js:27]: - [ ] Kindly consider following the order of sections as given in the Figma  design to make your design the the expected one cause currently this section i being displayed after the footer

File: script.js [hunk 3]
```diff
@@ -21,3 +21,508 @@ menuLinks.forEach((link) => {
...
+
+  project1Div.appendChild(imgProject1Div);
+  project1Div.appendChild(workTitleContentTagDiv);
+  section.appendChild(project1Div);
+  const buttonWork1DesktopDiv = document.createElement('div');
+  buttonWork1DesktopDiv.className = 'button-work1-desktop';
+
+  const buttonWork1 = document.createElement('button');
+  buttonWork1.className = 'button-work1';
+  buttonWork1.textContent = 'See project';
```
Comment (divinecharlotte) [inline script.js:104]: - [ ] Kindly consider making sure that this button when clicked the expected pop up appears cause currently it's not being displayed and that will help you to meet the below requirement as  expected
> When the user clicks (or taps) the button to check project details, the popup with details about the project appears

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Spacelama587   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Branch named well👍 
- Descriptive Pr 👍 

 




 

## Required Changes ♻️

_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -1116,3 +1110,200 @@ form {
...
+.image-text-container img {
+  max-width: 50%;
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: 10% auto 0;
+  position: relative;
+}
```
Comment (AbelG101) [inline style.css:1138]: - [ ] [OPTIONAL] I'd highly suggest you work on the design for the popup as there's a discrepancy between your design and the original design as you can see below 👇
- Your design:
  image
- Figma design:
  image

**Review by AbelG101 — APPROVED** [review]
# Hi @Spacelama587 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**